runtime/doc/starting.txt

```diff
@@ -373,24 +373,24 @@ accordingly.  Vim proceeds in this order:
 3. Execute Ex commands, from environment variables and/or files
 	An environment variable is read as one Ex command line, where multiple
 	commands must be separated with '|' or "<NL>".
-								*vimrc* *exrc*
+						    *init.vim* *vimrc* *exrc*
 	A file that contains initialization commands is called a "vimrc" file.
 	Each line in a vimrc file is executed as an Ex command line.  It is
 	sometimes also referred to as "exrc" file.  They are the same type of
 	file, but "exrc" is what Vi always used, "vimrc" is a Vim specific
-	name.  Also see |vimrc-intro|.
+	name, "init.vim" is Neovim specific location for vimrc file.  Also see 
+	|vimrc-intro|.
 
 	Places for your personal initializations:
-		Unix		$HOME/.vimrc or $HOME/.vim/vimrc
-		MS-Windows	$HOME/_vimrc, $HOME/vimfiles/vimrc
-				or $VIM/_vimrc
+		Unix		$XDG_CONFIG_HOME/.config/nvim/init.vim
+				(default for $XDG_CONFIG_HOME is ~/.config)
 
 	The files are searched in the order specified above and only the first
 	one that is found is read.
 
-	RECOMMENDATION: Put all your Vim configuration stuff in the
-	$HOME/.vim/ directory ($HOME/vimfiles/ for MS-Windows). That makes it
-	easy to copy it to another system.
+	RECOMMENDATION: Put all your Vim configuration stuff in the 
+	$HOME/.config/nvim/ directory. That makes it easy to copy it to 
```

Comment:
`$XDG_CONFIG_HOME` was just  introduced above, so I think it should be used here instead of `$HOME/.config`.